azure/v31.1.1/announcement.md

```diff
@@ -0,0 +1,3 @@
+**Workload cluster release v31.1.1 for Azure is available**.
+
+Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-capz/releases/azure-31.1.1).
```

Comment:
```suggestion
Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-azure/releases/azure-31.1.1).
```

Reply:
Eh, this is how they are getting generated from `devctl`. But apparently `devctl` is wrong. Gonna fix that.